x-pack/legacy/plugins/siem/public/components/timeline/footer/last_updated.tsx

```diff
@@ -37,46 +33,38 @@ export const Updated = pure<{ date: number; prefix: string; updatedAt: number }>
 
 Updated.displayName = 'Updated';
 
-export class LastUpdatedAt extends React.PureComponent<LastUpdatedAtProps, LastUpdatedAtState> {
-  public readonly state = {
-    date: Date.now(),
-  };
-  private timerID?: NodeJS.Timeout;
+const prefix = ` ${i18n.UPDATED} `;
 
-  public componentDidMount() {
-    this.timerID = setInterval(() => this.tick(), 10000);
-  }
+export const LastUpdatedAt = React.memo<LastUpdatedAtProps>(({ compact = false, updatedAt }) => {
+  const [date, setDate] = useState<number>(Date.now());
+  const timerID = useRef<NodeJS.Timeout | null>(null);
```

Comment:
This is using a `NodeJS.Timeout` when it should instead be using the browser types. The NodeJS timer and type script types are very different from the front end types.

Reply:
whats the browser type look like @FrankHassanabad

Reply:
i guess doesn't matter cuz i'm eliminating this var, just for the future??